includes/class.llms.student.query.php

```diff
@@ -417,16 +419,18 @@ private function sql_status_in( $column = 'status' ) {
 	}
 
 	/**
-	 * Generate an SQL subquery for the dynamic status or date values in the main query
+	 * Generate an SQL subquery for the dynamic status or date values in the main query.
```

Comment:
I would change this to reflect the new "ability" added :D

Reply:
Is this good?
```Generate an SQL subquery for the meta key in the main query.```

Reply:
I think so.